css-fonts/styles.css

```diff
@@ -0,0 +1,4 @@
+* {
+  font-family: lato, Montserrat,sans-serif ;
+  line-height: 2em ;
+}
```

Comment:
For this one, I know instructions were not super clear on this (now that I am reading it) but we want to pick two fonts from Google fonts and apply each to heading and other body, so one for each.